### ISSUE TITLE
Recommended to Install as root/non-root

### ISSUE THREAD

**Issue by bilogic** [mail-in-a-box/mailinabox#2452]
![image](https://github.com/user-attachments/assets/56d4588b-b658-4927-8360-d32bfe171a0e)

Possible to explicity state which is the recommended installation approach on the homepage near this part?
I know `sudo` implies installing as non-root, but it is inferred and some may not know.

Many have asked before too, e.g. https://discourse.mailinabox.email/t/should-miab-be-installed-from-root-or-non-root-user/4648

**Comment by bilogic** [comment]
I just realized this might be better transferred over to https://github.com/mail-in-a-box/mailinabox.email

And I'm ready to file a PR.

**Comment by JoshData** [comment]
I don't think it matters as long as you do the same thing whenever you run the script again in the future.

**Comment by myfirstnameispaul** [comment]
A user can delete files (and empty directories) created with `sudo` and owned by `root` that are in the user home directory, but not in subdirectories. I haven't explored this aspect of permissions before, not sure if it is relevant.

**Comment by bilogic** [comment]
Sure, my request is to have this documented so that I don't have to ponder/try to recall the next round.
